react/features/presence-status/components/PresenceLabel.js

```diff
@@ -124,8 +125,9 @@ function _mapStateToProps(state, ownProps) {
     const participant = getParticipantById(state, ownProps.participantID);
 
     return {
-        _presence:
-            (participant && participant.presence) || ownProps.defaultPresence
+        _presence: presenceStatusDisabled() ? ''
```

Comment:
Rather than `''` this shoiuld be `null`, right?

Also, since we are here, the sencod part can be written as: `participant?.presence ?? ownProps.defaultPresence` (that combines optional chaining with nullish coalescing, for extra awesomeness points 😎 )

Reply:
`_presence` is a string. I think we should try to be type consistent if we already use `flow`.

Reply:
Good point! Keep it `''` indeed.